fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -539,7 +539,7 @@ protected Integer getResourceTypeIdFromCaches(String resourceType) {
     }
 
     @Override
-    public Resource insert(Resource resource, List<ExtractedParameterValue> parameters, ParameterDAO parameterDao)
+    public Resource insert(Resource resource, List<ExtractedParameterValue> parameters, String parameterHashB64, ParameterDAO parameterDao)
```

Comment:
Is parameterHashB64 supposed to be used in this method?

Reply:
It is now.